src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/edit_index_pattern.js

```diff
@@ -140,4 +143,43 @@ uiModules.get('apps/management')
       $scope.indexPattern.timeFieldName = field.name;
       return $scope.indexPattern.save();
     };
+
+    $scope.tryToRenderScriptedFieldsTable = function () {
+      if ($state.tab === 'scriptedFields') {
```

Comment:
This file and a couple of others are just so bound up in Angular.  Since we are supposed to be running away from Angular anyway, I wonder if it makes sense to take this Reactification work farther ?

Reply:
Yea I agree, but I think the focus should be on the tables for now to ensure those can go out sooner rather than later. Once the tables are in, I want to take a pass on the shell part (`edit_index_pattern`)

Reply:
I guess I just worry about people looking at these squashed merged changes and taking them as patterns.

Reply:
And maybe there are bugs in all this glue code.  Just seems better to get out of this to me.

Reply:
Yea I don't want that to happen, but converting this to React is probably more work than it seems because this is a shell that renders [angular directives](https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/edit_index_pattern.html#L141) and getting those to work within a React shell seems complicated too. 

What if I renamed the variables/functions in here to names that directly state to not use this and it's just a stopgap?